src/externals.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Re-exports commonly used modules:
+ * * Adds [`ethjsUtil`](https://github.com/ethjs/ethjs-util) methods.
+ * * Exports [`BN`](https://github.com/indutny/bn.js), [`rlp`](https://github.com/ethereumjs/rlp), [`secp256k1`](https://github.com/cryptocoinjs/secp256k1-node/)`.
+ * @packageDocumentation
+ */
+
+const ethjsUtil = require('ethjs-util')
+import * as secp256k1 from 'secp256k1'
+import * as BN from 'bn.js'
+import * as rlp from 'rlp'
+
+/**
+ * [`ethjsUtil`](https://github.com/ethjs/ethjs-util)
+ */
+Object.assign(exports, ethjsUtil)
+
+/**
+ * [`BN`](https://github.com/indutny/bn.js)
+ */
+export { BN }
+
+/**
+ * [`rlp`](https://github.com/ethereumjs/rlp)
+ */
+export { rlp }
+
+/**
+ * [`secp256k1`](https://github.com/cryptocoinjs/secp256k1-node/)
+ */
+export { secp256k1 }
```

Comment:
Nice to have this in a separate file now, this generally make library exports easier to grasp and more explicit, e.g. to have this new `externals.spec.ts` test file along is a great addition.

I first thought if it would be necessary to hide this respectively just have this for documentation purposes and recommend (e.g. by stating in the docs) to use the `index.ts` exposure for importing the re-exports as a user. Second thought lead me to think that this is not necessary and it is very much ok to have this new `externals` entrypoint as an alternative. This will likely stay pretty stable and remains fully backwards-compatible.